functions/nodejs-apps.bash

```diff
@@ -346,7 +344,10 @@ zigbee2mqtt_setup() {
   echo "OK"
   
   echo -n "$(timestamp) [openHABian] Setting up Zigbee2MQTT service... "
-  if ! cond_redirect install -m 644 "${BASEDIR:-/opt/openhabian}"/includes/zigbee2mqtt/zigbee2mqtt.service /etc/systemd/system/; then echo "FAILED (install service)"; return 1; fi
+
+  sed -e "s|%user%|${username:-openhabian}|g" "${BASEDIR:-/opt/openhabian}/includes/zigbee2mqtt/zigbee2mqtt.service" | sudo -u "${username:-openhabian}" dd status=none of=/tmp/zigbee2mqtt.service
```

Comment:
dd? why ??

Reply:
It takes the input from sed and with user openhabian writes to another location (without conversion of course). Otherwise I would need another chmod. If you know a better way tell me.  
It's just the same as 9 lines above for configuration.yaml.

Reply:
what's wrong about another chown ? That's way more transparent.
Btw you can also do sed -i to edit some file in place so rights are kept.

Reply:
I know. But the template in the includes-directory should not be changed.
dd duplicates from standard-in to standard-out, so I got everything done in one pipe. Why is it not transparent?
As I say it's just the same as above. Feel free to change if you like.

Reply:
Nah, I'm just the reviewer.
Code that isn't self-explaining and unnecessarily complex to understand isn't as good as simple, straightforward code even if that's longer.

Reply:
then I change the existing one, too.
I think it's fine now.
Don't forget to make a release including the raspberry-imager changes. Otherwise we drop out of raspberry imager, because the json-file with the new flags cannot be downloaded.